marconi-chain-index/src/Marconi/ChainIndex/Indexers/Utxo.hs

```diff
@@ -215,93 +225,81 @@ instance ToJSON Utxo where
             , "txIndexInBlock"    .= (u ^. txIndexInBlock)
             ]
 
-data ChainPointRow
-    = ChainPointRow { _cpSlotNo :: C.SlotNo, _cpBlockHash :: C.Hash C.BlockHeader }
-    deriving (Show, Eq, Ord, Generic)
-
-$(makeLenses ''ChainPointRow)
-
-toChainPointRow :: C.ChainPoint -> Maybe ChainPointRow
-toChainPointRow cp = ChainPointRow <$> C.chainPointToSlotNo cp <*> C.chainPointToHeaderHash cp
-
-instance FromJSON ChainPointRow where
-    parseJSON (Object v)
-        = ChainPointRow
-            <$> v .: "slotNo"
-            <*> v .: "blockHeaderHash"
-    parseJSON _ = mempty
-
-instance ToJSON ChainPointRow where
-    toJSON c
-        = object
-        [ "slotNo" .= view cpSlotNo c
-        , "blockHeaderHash" .= view cpBlockHash c
-        ]
-
-getChainPoint :: ChainPointRow -> C.ChainPoint
-getChainPoint cp = C.ChainPoint (cp ^. cpSlotNo) (cp ^. cpBlockHash)
-
 data SpentInfo
     = SpentInfo
-    { _siSpentPoint :: ChainPointRow
-    , _siSpentTxId  :: C.TxId
+    { _siSlotNo          :: C.SlotNo
+    , _siBlockHeaderHash :: C.Hash C.BlockHeader
+    , _siSpentTxId       :: C.TxId
     } deriving (Show, Eq, Ord, Generic)
 
 $(makeLenses ''SpentInfo)
 
 data UtxoRow = UtxoRow
-  { _urUtxo          :: !Utxo
-  , _urCreationPoint :: !ChainPointRow
-  , _urSpentInfo     :: !(Maybe SpentInfo)
+  { _urUtxo            :: !Utxo
+  , _urSlotNo          :: !C.SlotNo
```

Comment:
I like the old names: `urCreationSlotNo` and `urCreationBlockHeaderHash`.

Reply:
9e799af49d9b82b99ced8dde8fcd0e0c46b4eb4f